src/ensembl/src/shared/components/view-in-app/ViewInApp.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { ImageButton } from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as EntityViewerIcon } from 'static/img/launchbar/entity-viewer.svg';
+
+import { Status } from 'src/shared/types/status';
+
+import styles from './ViewInApp.scss';
+
+export const Apps = {
+  gb: {
+    tooltip: 'Genome Browser',
+    icon: BrowserIcon
+  },
+  ev: {
+    tooltip: 'Entity Viewer',
+    icon: EntityViewerIcon
+  }
```

Comment:
Ew. I would definitely prefer `genomeBrowser` and `entityViewer` to `gb` and `ev`. Would love to hear what others have to say.